openlcs/authentication/admin.py

```diff
@@ -12,7 +12,6 @@ class RedHatProfileInline(admin.StackedInline):
     can_delete = True
     verbose_name_plural = 'RedHatProfile'
     fk_name = 'user'
-    fields = ('sub', 'roles', 'full_name')
```

Comment:
Here only remove `roles`, keep `sub` and `full_name`

Reply:
Updated. Thanks for your review :)